src/styles/partials/_theme.scss

```diff
@@ -0,0 +1,169 @@
+body{
+	color: $primary-color;
+	margin: 0;
+	padding: 0;
+	width: 100vw;
+	min-height: 100vh;
+	background: #fff;
+	overflow-x: hidden;
+}
+
+a {
+	color: $link-color;
+}
+
+a:hover {
+	color: $link-hover-color;
+	text-decoration: underline;
+	cursor: pointer;
+}
+
+hr {
+	margin-top: 20px;
+	margin-bottom: 20px;
+	border: 0;
+	border-top: 1px solid $line-color;
+}
+
+
+
+/****HOME PAGE SEARCH BOX ELEMENTS****/
```

Comment:
I forgot to remove HOME PAGE from comments. These styles correspond to the basic styling of both the search input at the home page and also on the header of search results page. Hence global. Only the basic styles regarding display and flow are here. The customisation according to the component are present in their component.scss file.